### PR TITLE
replaced points_concat_filter launch with point_cloud_fusion_node launch

### PR DESCRIPTION
**Pull request by john-chrosniak**
# PR Details
## Description

This PR updates the launch scripts for CARMA freight to launch the [point_cloud_fusion_node](https://github.com/usdot-fhwa-stol/autoware.auto/tree/develop/AutowareAuto/src/perception/filters/point_cloud_fusion_nodes) package, Autoware.Auto's ROS2 version of the [points_concat_filter](https://github.com/usdot-fhwa-stol/autoware.ai/tree/carma-develop/core_perception/points_preprocessor/nodes/points_concat_filter) package in Autoware.AI. This is configured to fuse the point clouds from the two LiDAR sensors onboard the CARMA freight trucks.

Related PRs:
- [usdotfhwastol/autoware.auto #21](https://github.com/usdot-fhwa-stol/autoware.auto/pull/21)
- [usdotfhwastol/carma-velodyne-lidar-driver #111](https://github.com/usdot-fhwa-stol/carma-velodyne-lidar-driver/pull/111)

## Related Issue

[CAR-5961](https://usdot-carma.atlassian.net/browse/CAR-5961)

## Motivation and Context

Currently, freightliner trucks equipped with CARMA Platform have two velodyne lidars (compared to the single velodyne lidar on our passenger vehicles equipped with CARMA Platform), and the autoware.ai ‘[points_concat_filter](https://github.com/usdot-fhwa-stol/autoware.ai/tree/carma-develop/core_perception/points_preprocessor/nodes/points_concat_filter)’ node (a ROS 1 Noetic node) is used to concatenate these two point clouds into a single point cloud to be used by the rest of the system. As part of the overall migration from ROS 1 Noetic to ROS 2 Foxy for CARMA Platform, the system should leverage the ROS 2 Foxy version of this node made available in Autoware.Auto.

## How Has This Been Tested?

This was tested by launching CARMA Platform on the freight truck and verifying that `point_cloud_fusion_nodes` fused the scans from the two velodyne sensors and published the result with the correct topic name and frame id.

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

[CAR-5961]: https://usdot-carma.atlassian.net/browse/CAR-5961?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: freightliner_cascadia_2012_dot_10002/drivers.launch [hunk 1]
```diff
@@ -55,18 +55,10 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="avt_vimba_camera"               value="$(eval ('avt_vimba_camera' in arg('drivers').split()) and not arg('mock_camera'))"/>
 
   <!-- PACMOD Controller Driver Node -->
-  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_10002_driver.launch">
+  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_80550_driver.launch">
```
Comment (JonSmet) [inline freightliner_cascadia_2012_dot_10002/drivers.launch:58]: Let's revert this change since truck 10002 (its license plate number) will use `ssc_pm_cascadia_dot_10002_driver.launch`

File: freightliner_cascadia_2012_dot_10003/drivers.launch [hunk 2]
```diff
@@ -55,18 +55,10 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="avt_vimba_camera"               value="$(eval ('avt_vimba_camera' in arg('drivers').split()) and not arg('mock_camera'))"/>
 
   <!-- PACMOD Controller Driver Node -->
-  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_10003_driver.launch">
+  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_80550_driver.launch">
```
Comment (JonSmet) [inline freightliner_cascadia_2012_dot_10003/drivers.launch:58]: Let's revert this change since truck 10003 (its license plate number) will use `ssc_pm_cascadia_dot_10003_driver.launch`

File: freightliner_cascadia_2012_dot_10004/drivers.launch [hunk 3]
```diff
@@ -55,18 +55,10 @@ If not using simulated drivers they are activated if the respective mock argumen
   <arg name="avt_vimba_camera"               value="$(eval ('avt_vimba_camera' in arg('drivers').split()) and not arg('mock_camera'))"/>
 
   <!-- PACMOD Controller Driver Node -->
-  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_10004_driver.launch">
+  <include if="$(arg ssc_interface_wrapper)" file="$(find ssc_interface_wrapper)/launch/ssc_pm_cascadia_dot_80550_driver.launch">
```
Comment (JonSmet) [inline freightliner_cascadia_2012_dot_10004/drivers.launch:58]: Let's revert this change since truck 10004 (its license plate number) will use `ssc_pm_cascadia_dot_10004_driver.launch`

File: freightliner_cascadia_2012_dot_10004/drivers.launch.py [hunk 4]
```diff
@@ -94,7 +94,7 @@ def generate_launch_description():
                             'frame_id' : 'velodyne_1',
                             'device_ip' : '192.168.1.201',
                             'port' : '2368',
-                            'gps_time' : 'True'
+                            'gps_time' : 'False'
```
Comment (JonSmet) [inline freightliner_cascadia_2012_dot_10004/drivers.launch.py:97]: Just wanted to confirm whether this change for the `gps_time` parameter in this file and in 80550's `drivers.launch.py` was required, and if so, do we know why?

**Review by JonSmet — CHANGES_REQUESTED** [review]

File: freightliner_cascadia_2012_dot_10002/drivers.launch (hunk 1 above)
Comment (john-chrosniak) [reply freightliner_cascadia_2012_dot_10002/drivers.launch]: Fixed!

File: freightliner_cascadia_2012_dot_10003/drivers.launch (hunk 2 above)
Comment (john-chrosniak) [reply freightliner_cascadia_2012_dot_10003/drivers.launch]: Fixed!

File: freightliner_cascadia_2012_dot_10004/drivers.launch (hunk 3 above)
Comment (john-chrosniak) [reply freightliner_cascadia_2012_dot_10004/drivers.launch:58]: Fixed!

File: freightliner_cascadia_2012_dot_10004/drivers.launch.py (hunk 4 above)
Comment (john-chrosniak) [reply freightliner_cascadia_2012_dot_10004/drivers.launch.py]: This change was a relic of debugging LiDAR message timestamps and was not needed, just reverted this

File: freightliner_cascadia_2012_dot_10004/drivers.launch (hunk 3 above)
Comment (john-chrosniak) [reply freightliner_cascadia_2012_dot_10004/drivers.launch]: Forgot to push this but it is now actually fixed

**Review by JonSmet — APPROVED** [review]
